<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountWindow</class>
 <widget class="QMainWindow" name="AccountWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="contLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>cont</string>
    </property>
   </widget>
   <widget class="QLabel" name="balanceLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>disponibil in cont</string>
    </property>
   </widget>
   <widget class="QLabel" name="operatiiLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Operatii disponibile:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="extragereButon">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Extrage</string>
    </property>
   </widget>
   <widget class="QPushButton" name="depunereButon">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Depunere</string>
    </property>
   </widget>
   <widget class="QPushButton" name="transferButon">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Transfera</string>
    </property>
   </widget>
   <widget class="QPushButton" name="donateButon">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Doneaza bani copiilor saraci</string>
    </property>
   </widget>
   <widget class="QLabel" name="msgLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>490</y>
      <width>461</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="backButon">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Alege alt cont</string>
    </property>
   </widget>
   <widget class="QLabel" name="notifTransLabel">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>260</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="transButon">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>300</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Vezi tranzactii</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
